ush/parsing_namelists_FV3.sh

```diff
@@ -422,6 +422,11 @@ if [[ ${DO_LAND_PERT:-"NO"} = "YES" ]]; then
 EOF
 fi
 
+# Close &gfs_physics_nml section
```

Comment:
Is there an issue open for this bug that should be added to the PR?

Reply:
No issue exists for this bug.  It is being pulled from PR #1865